src/packages/@ncigdc/containers/explore/ClinicalAggregations.tsx

```diff
@@ -280,7 +280,7 @@ const enhance = compose(
       dispatch,
       notifications,
     }: IClinicalProps): any => {
-      return [
+       return (<Column>
```

Comment:
Maybe this could be a Fragment instead? That way we don't add more HTML elements/markup.